js/main.js

```diff
@@ -314,6 +357,9 @@ function deleteEntry(targetMovie) {
     }
     resetAddButton();
   }
+  if (!data.entries.length > 0) {
```

Comment:
This should also work shortened

```JAVASCRIPT
if (data.entries.length) {
```

If length is `0` that is false so the code block won't execute but if it is 1 or greater that is truthy so the code block will run.